gtk/src/auto/mod.rs

```diff
@@ -1142,3 +1142,167 @@ pub mod traits {
     pub use super::window::GtkWindowExt;
     pub use super::window_group::WindowGroupExt;
 }
+#[doc(hidden)]
+pub mod builders {
+    pub use super::about_dialog::AboutDialogBuilder;
```

Comment:
Shouldn't this be re-exported from `src/lib.rs` like

```rust
pub mod builders {
    pub use foo::ManualBuilder;
    pub use auto::builders::*;
}
```

?

Reply:
Oh you do that here already. This commit is not just regenerating but also doing some manual changes.

Reply:
Ah yeah, let me quickly split those